src/Sulu/Bundle/AdminBundle/Resources/js/components/ImageRectangleSelection/ImageRectangleSelection.js

```diff
@@ -23,10 +23,18 @@ class ImageRectangleSelection extends React.Component<Props> {
     image: Image;
     @observable imageLoaded = false;
 
-    naturalHorizontalToScaled = (h: number) => h * this.imageResizedWidth / this.image.naturalWidth;
-    scaledHorizontalToNatural = (h: number) => h * this.image.naturalWidth / this.imageResizedWidth;
-    naturalVerticalToScaled = (v: number) => v * this.imageResizedHeight / this.image.naturalHeight;
-    scaledVerticalToNatural = (v: number) => v * this.image.naturalHeight / this.imageResizedHeight;
+    naturalHorizontalToScaled = (h: number) => {
+        return Math.max(h * this.scaledImageWidth / this.image.naturalWidth, 0);
+    };
+    scaledHorizontalToNatural = (h: number) => {
+        return Math.min(h * this.image.naturalWidth / this.scaledImageWidth, this.image.naturalWidth);
```

Comment:
if the image is scaled down a lot in the crop overlay, `h * this.image.naturalWidth / this.scaledImageWidth` might result in a number bigger than `this.image.naturalWidth`